packages/medusa/src/api/middlewares/error-handler.ts

```diff
@@ -51,7 +53,6 @@ export default () => {
         break
       default:
         errObj.code = "unknown_error"
-        errObj.message = "An unknown error occurred."
```

Comment:
question: is this replaced somewhere?

Reply:
it just not overriding line 23 in fact 😂